target/linux/generic/pending-5.4/762-net-dsa-mt7530-enable-jumbo-frame.patch

```diff
@@ -0,0 +1,69 @@
+From 953e2756402d30aad4b4ea7fce3870ad75e11939 Mon Sep 17 00:00:00 2001
+From: DENG Qingfang <dqfext@gmail.com>
+Date: Thu, 9 Apr 2020 23:34:18 +0800
+Subject: [PATCH] net: dsa: mt7530: enable jumbo frame
+
+Enable jumbo frame for MT7530
+It allows frames up to 15 KB (including FCS)
+
+Signed-off-by: DENG Qingfang <dqfext@gmail.com>
```

Comment:
This implementation is already rejected upstream so I'm rejecting this patch as well.